### ISSUE TITLE
`component` argument not recognized for glmmTMB models

### ISSUE THREAD

**Issue by strengejacke** [bbolker/broom.mixed#32]
``` r
library(glmmTMB)
library(broom.mixed)

m1 <- glmmTMB(count~ mined + (1|site), 
              zi=~mined, 
              family=poisson, data=Salamanders)

broom.mixed::tidy(m1, effects = "fixed", component = "cond")
#> # A tibble: 4 x 7
#>   effect component term        estimate std.error statistic  p.value
#>   <chr>  <chr>     <chr>          <dbl>     <dbl>     <dbl>    <dbl>
#> 1 fixed  cond      (Intercept)   0.0879     0.233     0.377 7.06e- 1
#> 2 fixed  cond      minedno       1.14       0.246     4.64  3.50e- 6
#> 3 fixed  zi        (Intercept)   1.14       0.235     4.85  1.26e- 6
#> 4 fixed  zi        minedno      -1.74       0.262    -6.63  3.44e-11

broom.mixed::tidy(m1, effects = "fixed", component = "zi")
#> # A tibble: 4 x 7
#>   effect component term        estimate std.error statistic  p.value
#>   <chr>  <chr>     <chr>          <dbl>     <dbl>     <dbl>    <dbl>
#> 1 fixed  cond      (Intercept)   0.0879     0.233     0.377 7.06e- 1
#> 2 fixed  cond      minedno       1.14       0.246     4.64  3.50e- 6
#> 3 fixed  zi        (Intercept)   1.14       0.235     4.85  1.26e- 6
#> 4 fixed  zi        minedno      -1.74       0.262    -6.63  3.44e-11
```

<sup>Created on 2018-10-04 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by bbolker** [comment]
hmm. will fix, but now I wonder whether it's better to have a `component` argument at all, or just let users `filter()` to get whichever components they actually want to keep ... I guess it's worth it for cases where e.g. we're asking for profile confidence intervals on selected component parameters ...

**Comment by bbolker** [comment]
I think this one should be fixed.

**Comment by bbolker** [comment]
fixed in 2db9ab340248
